### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by danieldk**
Documentation updates for changes that I made since the last release.

### CODE REVIEW

File: website/docs/api-backends.md [hunk 1]
```diff
@@ -152,11 +152,12 @@ Given an `(M, N)` sequence of vectors, return an `(M, N*(nW*2+1))` sequence. The
...
 vectors onto each column in the sequence, to extract a window of features.
 
-| Argument    | Type              | Description                                                       |
-| ----------- | ----------------- | ----------------------------------------------------------------- |
-| `seq`       | <tt>Floats2d</tt> | The original sequence .                                           |
-| `nW`        | <tt>int</tt>      | The window size.                                                  |
-| **RETURNS** | <tt>Floats2d</tt> | The created sequence containing preceding and succeeding vectors. |
+| Argument    | Type                      | Description                                                       |
+| ----------- | ------------------------- | ----------------------------------------------------------------- |
+| `seq`       | <tt>Floats2d</tt>         | The original sequence .                                           |
```
Comment (svlandeg) [inline website/docs/api-backends.md:157]: Old typo
```suggestion
| `seq`       | <tt>Floats2d</tt>         | The original sequence.                                           |
```

File: website/docs/api-backends.md [hunk 2]
```diff
@@ -152,11 +152,12 @@ Given an `(M, N)` sequence of vectors, return an `(M, N*(nW*2+1))` sequence. The
...
+| `nW`        | <tt>int</tt>              | The window size.                                                  |
+| `lengths`   | <tt>Optional[Ints1d]</tt> | Sequence lengths, introduces padding around sequences.            |
```
Comment (svlandeg) [inline website/docs/api-backends.md:159]: `lengths` is a kwd-only arg, right? You can add a divider `_keyword-only_`

File: website/docs/api-backends.md [hunk 3]
```diff
@@ -172,11 +173,12 @@ The reverse/backward operation of the `seq2col` function: calculate the gradient
...
-| Argument    | Type              | Description                        |
-| ----------- | ----------------- | ---------------------------------- |
-| `dY`        | <tt>Floats2d</tt> | Gradient of the output sequence.   |
-| `nW`        | <tt>int</tt>      | The window size.                   |
-| **RETURNS** | <tt>Floats2d</tt> | Gradient of the original sequence. |
+| Argument    | Type                      | Description                                            |
+| ----------- | ------------------------- | ------------------------------------------------------ |
+| `dY`        | <tt>Floats2d</tt>         | Gradient of the output sequence.                       |
+| `nW`        | <tt>int</tt>              | The window size.                                       |
+| `lengths`   | <tt>Optional[Ints1d]</tt> | Sequence lengths, introduces padding around sequences. |
```
Comment (svlandeg) [inline website/docs/api-backends.md:180]: Same comment: kwd-only arg

File: website/docs/api-backends.md [hunk 4]
```diff
@@ -661,13 +664,14 @@ Calculate the softmax function. The resulting array will sum up to 1.
...
-| _keyword-only_ |                   |                                         |
-| `axis`         | <tt>int</tt>      | The dimension that was normalized over. |
-| **RETURNS**    | <tt>FloatsXd</tt> | The gradients of the input array.       |
+| Argument       | Type              | Description                                          |
+| -------------- | ----------------- | ---------------------------------------------------- |
+| `Y`            | <tt>FloatsXd</tt> | Output array.                                        |
+| `dY`           | <tt>FloatsXd</tt> | Gradients of the output array.                       |
+| _keyword-only_ |                   |                                                      |
+| `axis`         | <tt>int</tt>      | The dimension that was normalized over.              |
+| `temperature`  | <tt>float</tt>    | The value to divide the unnormalized probabilies by. |
```
Comment (svlandeg) [inline website/docs/api-backends.md:673]:
```suggestion
| `temperature`  | <tt>float</tt>    | The value to divide the unnormalized probabilities by. |
```

File: website/docs/api-backends.md [hunk 5]
```diff
@@ -643,13 +645,14 @@ Calculate the derivative of the `tanh` function.
...
-| `inplace`      | <tt>bool</tt>     | If `True`, the array may be modified in place. |
-| `axis`         | <tt>int</tt>      | The dimension to normalize over.               |
-| **RETURNS**    | <tt>FloatsXd</tt> | The normalized output values.                  |
+| Argument       | Type              | Description                                          |
+| -------------- | ----------------- | ---------------------------------------------------- |
+| `x`            | <tt>FloatsXd</tt> | The input values.                                    |
+| _keyword-only_ |                   |                                                      |
+| `inplace`      | <tt>bool</tt>     | If `True`, the array may be modified in place.       |
+| `axis`         | <tt>int</tt>      | The dimension to normalize over.                     |
+| `temperature`  | <tt>float</tt>    | The value to divide the unnormalized probabilies by. |
```
Comment (svlandeg) [inline website/docs/api-backends.md:654]:
```suggestion
| `temperature`  | <tt>float</tt>    | The value to divide the unnormalized probabilities by. |
```
(will require reformat)

**Review by svlandeg — COMMENTED** [review]
Thanks for adding these! Had a few small comments.

File: website/docs/api-backends.md (hunk 2 above)
Comment (danieldk) [reply website/docs/api-backends.md]: Ah, that's nice! Fixed.

**Review by svlandeg — APPROVED** [review]
Great. If this is the final docs change before the release, we can merge it into the docs branch and let the website build, if we're otherwise good to go for the release :-)

**Comment by danieldk** [comment]
> If this is the final docs change before the release, we can merge it into the docs branch and let the website build, if we're otherwise good to go for the release :-)

@kadarakos is working on another PR with docs for the new activations.

**Pull request merged**